### PR TITLE
Features cqf 984 122 test

### PR DESCRIPTION
**Pull request by TychonautVII**
Parameterized Mat Measures Tests

**Description**

Wanted to put in a quick PR for a slight refactor of some of the Mat Measures Tests, to see if you're alright with it. If you're cool with this general approach, it will make it faster to add in all the other measures. I've also added in the measure for the specific ticket that I picked up, now that I have access to the google drive with the measures

- Issues: 
- https://tracker.esacinc.com/browse/CQF-984
- https://tracker.esacinc.com/browse/CQF-1027

- [X] I've read the contribution guidelines 
- [X] Code compiles without errors
- [X] Tests are created / updated
- [N/A ] Documentation is created / updated

By creating this PR you acknowledge that your contribution will be licensed under Apache 2.0

### CODE REVIEW

File: src/test/java/org/opencds/cqf/tooling/utilities/ECQMCreatorTest.java [hunk 1]
```diff
@@ -91,32 +92,21 @@ public void TestCMS125FHIR() {
...
+    @DataProvider(name="matMeasureLoader")
+    public static Object[][] matMeasures(){
+        return new Object [][] {
+            {"CMS125FHIR-v0-0-004-FHIR-4-0-1.json", "BreastCancerScreeningsFHIR"},
+            {"CMS104-v2-0-004-FHIR-4-0-1.json", "DischargedonAntithromboticTherapyFHIR"},
+        };
+    }
+
+
+    @Test(dataProvider = "matMeasureLoader")
```
Comment (TychonautVII) [inline src/test/java/org/opencds/cqf/tooling/utilities/ECQMCreatorTest.java:107]: Using a "DataProvider" like this is the core refactor I'm thinking about for these tests

File: scripts/branch.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+branch_desc=$1;
+branch_name="feature-${branch_desc}";
+
+git checkout master
+git pull
+git checkout -b $branch_name
+git push --set-upstream origin $branch_name
```
Comment (TychonautVII) [inline scripts/branch.sh:7]: this is just a helper script meant to support the branch naming structure we seem to be using

**Review by brynrhodes — CHANGES_REQUESTED** [review]
Discussed on sprint 8/30, submitter suggested rejecting in favor of a refactor that preserves test method per measure.

**Comment by brynrhodes** [comment]
Please preserve the method per measure

**Pull request merged**